src/ProjectService.Validation/CreateProjectValidator.cs

```diff
@@ -1,91 +1,179 @@
+          .WithMessage("Content can't be empty")
+          .Must(x => imageFormats.Contains(x.Extension))
+          .WithMessage("Wrong extension")
+          .Must(images => images.Name.Length < 150)
+          .WithMessage("Name's length must be less than 150 letters")
+          .Must(images =>
             {
-                RuleFor(project => project.Description)
-                    .MaximumLength(300)
-                    .WithMessage("Project description is too long.");
-            });
-
-            When(
-                news => news.DepartmentId.HasValue,
-                () =>
-                    RuleFor(news => news.DepartmentId)
-                        .Must(DepartmentId => DepartmentId != Guid.Empty)
-                        .WithMessage("Wrong type of department Id."));
+              try
+              {
+                var byteString = new Span<byte>(new byte[images.Content.Length]);
+                return Convert.TryFromBase64String(images.Content, byteString, out _);
+              }
+              catch
+              {
+                return false;
+              }
+            }).WithMessage("Wrong image content.");
+      });
+
+      When(project => !string.IsNullOrEmpty(project.ShortDescription?.Trim()), () =>
+      {
+        RuleFor(project => project.ShortDescription)
+          .MaximumLength(300)
+          .WithMessage("Project short description is too long.");
+      });
+
+      When(project => !string.IsNullOrEmpty(project.Description?.Trim()), () =>
+      {
+        RuleFor(project => project.Description)
+          .MaximumLength(300)
+          .WithMessage("Project description is too long.");
+      });
+
+      When(
+        news => news.DepartmentId.HasValue,
+        () =>
+          RuleFor(news => news.DepartmentId)
+            .Must(DepartmentId => DepartmentId != Guid.Empty)
+            .WithMessage("Wrong type of department Id."));
+    }
+
+    private bool CheckDepartmentExistence(Guid? departmentId)
+    {
+      if (!departmentId.HasValue)
+      {
+        return true;
+      }
+
+      string logMessage = "Department with id: {id} not found.";
+
+      try
+      {
+        var response = _rcCheckDepartmentsExistence.GetResponse<IOperationResult<ICheckDepartmentsExistence>>(
+          ICheckDepartmentsExistence.CreateObj(new List<Guid> { departmentId.Value })).Result;
+        if (response.Message.IsSuccess)
+        {
+          if (!response.Message.Body.DepartmentIds.Any())
+          {
+            return false;
+          }
+          return true;
+        }
+
+        _logger.LogWarning("Can not find department with this Id: {departmentId}: " +
+          $"{Environment.NewLine}{string.Join('\n', response.Message.Errors)}");
+      }
+      catch (Exception exc)
+      {
+        _logger.LogError(exc, logMessage);
+      }
+
+      return false;
+    }
+
+    private bool CheckUserExistence(List<Guid> userIds)
+    {
+      if (!userIds.Any())
+      {
+        return true;
```

Comment:
its strange that if we send empty list of userIds we return true. Its empty - its should not return that user exist.

Reply:
done